games/p.yaml

```diff
@@ -164,6 +164,28 @@
   updated: 2015-04-16
   url: http://pathological.sourceforge.net/
 
+- name: PCExhumed
+  type: remake
+  originals:
+  - Powerslave
+  repo: https://github.com/nukeykt/NBlood
+  url: http://pcex.retrohost.net/
+  development: very active
+  status: playable
+  lang:
+  - C++
+  framework:
+  - SDL
+  - SDL2
+  - OpenGL
+  license:
+  - GPL2
```

Comment:
Change `license` to `Custom`. Every Build engine game's license is custom due to Ken Silverman's buildlic.txt.